src/renderers/shared/reconciler/__tests__/ReactCompositeComponentNestedState-test.js

```diff
@@ -88,10 +88,12 @@ describe('ReactCompositeComponentNestedState-state', function() {
 
     var logger = mocks.getMockFunction();
 
-    var instance = React.render(
+    /*eslint-disable no-unused-expressions */
```

Comment:
It was triggering an `unused-var` warning. Regardless, the `void` change fixes it.